docs/application/tizen-studio/native-tools/manifest-text-editor.md

```diff
@@ -1210,6 +1213,65 @@ For more information on the relationship between the elements, see the [element
 <feature name="http://tizen.org/feature/fmradio">true</feature>
 ```
 
+### \<dependencies\> Element
+
+This element contains a set of packages which are have dependent to this Tizen package. The package can be installed when all dependencies are satisfied.
+
+For more information on the relationship between the elements, see the [element hierarchy](#hierarchy).
+
+**Occurrences:**
+
+- 1 (optional)
+
+**Expected children:**
+
+| Child element             | Occurrences          |
+|---------------------------|----------------------|
+| `<dependency>`            | 1 or more (optional) |
+
+**For example:**
+
+```xml
+<dependencies>
+   <dependency>.....</dependency>
+</dependencies>
+```
+
+#### \<dependency\> Element
+
+This element contains a required package for a Tizen package.
```

Comment:
This sentence is confusing. Kindly elaborate further.

Reply:
Please review my new sentence.